### PR TITLE
update several statements

### PR DESCRIPTION
**Pull request by bobotu**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3309

### CODE REVIEW

File: sql-statements/sql-statement-set-variable.md [hunk 1]
```diff
@@ -33,7 +43,12 @@ mysql> SHOW SESSION VARIABLES LIKE 'sql_mode';
 | sql_mode      | ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION |
 +---------------+-------------------------------------------------------------------------------------------------------------------------------------------+
 1 row in set (0.00 sec)
+```
 
+Update the value of `sql_mode` globally.
+Checking the value of `SQL_mode` after the update, you can see that the value of SESSION level has not been updated:
```
Comment (ran-huang) [inline sql-statements/sql-statement-set-variable.md:49]:
```suggestion
If you check the value of `SQL_mode` after the update, you can see that the value of `SESSION` level has not been updated:
```

File: sql-statements/sql-statement-show-variables.md [hunk 2]
```diff
@@ -25,85 +25,123 @@ This statement shows a list of variables for the scope of either `GLOBAL` or `SE
 
 ## Examples
 
+List all TiDB specific variables. For the meaning of them, refer to [TiDB Specific System Variables](/tidb-specific-system-variables.md).
```
Comment (ran-huang) [inline sql-statements/sql-statement-show-variables.md:28]:
```suggestion
List all TiDB specific variables. For detailed description, refer to [TiDB Specific System Variables](/tidb-specific-system-variables.md).
```

**Comment by bobotu** [comment]
PTAL @imtbkcat

**Pull request merged**